### ISSUE TITLE
Health endpoint

### ISSUE THREAD

**Issue by cohix** [suborbital/atmo#35]
Atmo should expose a health endpoint that can be used to determine the health of Atmo. This can include whether or not a bundle has been successfully loaded and Atmo is available to serve requests.

**Comment by cohix** [comment]
@zhangwinning sounds great! This will involve adding a handler to Atmo's router (which is generated by the coordinator - see `atmo/coordinator/coordinator.go`) that returns a 200ok response, indicating to the caller that Atmo is healthy. That handler should be mounted at the `/health` path.

**Comment by zhangwinning** [comment]
~Hi @cohix, I see there are rust [examples](https://github.com/suborbital/subo/tree/main/templates/project/helloworld) here, but I’m not familiar with rust, are there any js or ts examples?

**Comment by cohix** [comment]
@zhangwinning Yes! Atmo supports AssemblyScript (very similar to TypeScript), here's an example: https://github.com/suborbital/atmo/blob/main/example-project/as-graphql/src/lib.ts
